app/plugins/file-log.md

```diff
@@ -77,9 +77,21 @@ Every request will be logged separately in a JSON object separated by a new line
             "access-control-allow-origin": "*"
         }
     },
+    "tries": [
```

Comment:
Curious: why not? It should use the same serializer as the other ones, so yes. Maybe your understanding of the file-log plugin is that it doesn't log requests/response info?